daemon/exec/exec.go

```diff
@@ -0,0 +1,110 @@
+package exec
```

Comment:
This packaged was created and exec was moved to avoid a dependency on daemon, by container.